src/main/java/com/blogify/blogapi/endpoint/security/SecurityConf.java

```diff
@@ -81,6 +81,7 @@ protected void configure(HttpSecurity http) throws Exception {
                         new AntPathRequestMatcher("/ping"),
                         new AntPathRequestMatcher("/signup"),
                         new AntPathRequestMatcher("/users"),
+                        new AntPathRequestMatcher("/users/**"),
```

Comment:
/* but not /**

because, it replace just one agr "uid"

if it was like, /susers/{id}/.../.. , u cane use /**

Reply:
okay